### PR TITLE
Cleans up birthday field

### LINKED ISSUE #4006: updates to birthday field on profile
- [x] styling should match other field names
- [x] remove Format: 02/17/2015

image

### PR DESCRIPTION
**Pull request by itsjoekent**
Fixes #4006 

screen shot 2015-02-19 at 11 38 37 am

screen shot 2015-02-19 at 3 24 11 pm

@aaronschachter

### CODE REVIEW

File: lib/themes/dosomething/paraneue_dosomething/includes/form.inc [hunk 1]
```diff
@@ -277,6 +278,21 @@ function paraneue_dosomething_form_user_profile_form_alter(&$form, &$form_state,
   $form['account']['pass']['#process'] = $process;
 }
 
+function paraneue_dosomething_form_user_profile_after_build($form, &$form_state) {
+  $form['#weight'] = '-18';
+  $form['#attributes']['class'] = array('auth-twocol');
+  //$form['#theme_wrappers'] = array('form_element');
```
Comment (itsjoekent) [inline lib/themes/dosomething/paraneue_dosomething/includes/form.inc]: shit let me delete this

File: lib/themes/dosomething/paraneue_dosomething/includes/auth/register.inc [hunk 2]
```diff
@@ -107,17 +107,9 @@ function paraneue_dosomething_form_alter_register(&$form, &$form_state, $form_id
  *  Modified drupal form.
  */
 function paraneue_dosomething_register_after_build($form, &$form_state) {
+  $form['field_birthdate'][LANGUAGE_NONE][0] = paraneue_dosomething_form_user_profile_after_build($form['field_birthdate'][LANGUAGE_NONE][0], $form_state);
```
Comment (aaronschachter) [inline lib/themes/dosomething/paraneue_dosomething/includes/auth/register.inc:110]: Dang this is crazy. If it works, awesom ethough. Nice Drupalling

**Comment by aaronschachter** [comment]
Looks good. please just make sure to test both forms to make sure validates and saves ok then good to ship :+1:

File: lib/themes/dosomething/paraneue_dosomething/includes/auth/register.inc (hunk 2 above)
Comment (itsjoekent) [inline lib/themes/dosomething/paraneue_dosomething/includes/auth/register.inc:110]: Had to do it this way cause of the prefixing....

**Comment by itsjoekent** [comment]
Q/A checked per comment above ^

adding value

**Pull request merged**